spec/services/ccms/submitters/obtain_document_id_service_spec.rb

```diff
@@ -100,7 +107,7 @@ module Submitters # rubocop:disable Metrics/ModuleLength
               subject.call
               admin1_documents = SubmissionHistory.where(submission_id: submission.id).map(&:request).map { |x| x.scan(/ADMIN1/) }.flatten.count
               bstmt_documents = SubmissionHistory.where(submission_id: submission.id).map(&:request).map { |x| x.scan(/BSTMT/) }.flatten.count
-              expect(admin1_documents).to eq 3
+              expect(admin1_documents).to eq 4
```

Comment:
I thought the ticket said that gateway evidence should be tagged as `STATE` rather than `ADMIN1`?